de.dlr.sc.virsat.excel.ui/src/de/dlr/sc/virsat/excel/ui/wizards/ExportPage.java

```diff
@@ -190,9 +190,9 @@ private void createTemplateUI() {
 		label1 = new Label(composite1, SWT.NONE);
 		label1.setText("Template");
 
-		// destination name entry field
+        // destination name entry field

```

Comment:
The whole file is a whitespace mess. Let's maybe reformat everything in this file to tabs for now?